src/Core/Constants.cs

```diff
@@ -47,6 +47,9 @@ public static class Constants
         public const int TotpDefaultTimer = 30;
         public const int PasswordlessNotificationTimeoutInMinutes = 15;
         public const int KdfIterations = 600000;
+        public const int Argon2Iterations = 2;
+        public const int Argon2Memory = 19;
```

Comment:
In order to be more guessable I think it would be better to include the unit on the constant, if I understand correctly it would be `Argon2MemoryInMb`

Reply:
Updated in [4ff8a7e](https://github.com/bitwarden/mobile/pull/2328/commits/4ff8a7ecdb6b2e8ea4c22cb378ae9e3b226a2659)